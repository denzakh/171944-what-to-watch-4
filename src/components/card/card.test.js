import React from "react";
import renderer from "react-test-renderer";
import Card from "./card";

let testFilm = {
  title: `The Godfather`,
  poster: `https://m.media-amazon.com/images/M/MV5BM2MyNjYxNmUtYTAwNi00MTYxLWJmNWYtYzZlODY3ZTk3OTFlXkEyXkFqcGdeQXVyNzkwMjQ5NzM@._V1_SX300.jpg`,
  href: `/`
};

describe(`Card component`, ()=>{
  it(`should render Card component markup`, () => {
    const tree = renderer.create(
        <Card film={testFilm} onMouseEnter={()=>{}} onMouseLeave={()=>{}} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
