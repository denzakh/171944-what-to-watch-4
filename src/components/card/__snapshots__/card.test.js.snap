// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component should render Card component markup 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="The Godfather"
      height="175"
      src="https://m.media-amazon.com/images/M/MV5BM2MyNjYxNmUtYTAwNi00MTYxLWJmNWYtYzZlODY3ZTk3OTFlXkEyXkFqcGdeQXVyNzkwMjQ5NzM@._V1_SX300.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="/"
    >
      The Godfather
    </a>
  </h3>
</article>
`;
